staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -88,13 +90,18 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	// No new log formats should be added after generation is of flag options
 	logRegistry.Freeze()
+	fs.BoolVar(&o.LogSanitization, "experimental-logging-sanitization", false, `When enabled prevents logging of fields that tagged as sensitive (passwords, keys, tokens).
```

Comment:
PR title and more importantly release note need updating.
Do we need to print a warning that this is experimental still or is the flag name sufficient @dims?

Reply:
good point Ben, Let's add `[Experimental]` in the description at the beginning like this one please. https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/options/options.go#L383

Reply:
Name of flag in commit/PR/release note done.

Reply:
done